.env.example

```diff
@@ -1,15 +1,55 @@
+# Please modify the following environment variables accordingly.
+
+CTIMS_ENV=development
+CTIMS_API_VERSION=2.1
+
+
+#DATABASE_URL=mysql://ctims:ctims@localhost:3306/ctims
+#AES 256 key, it can be generated using https://cloak.47ng.com/key
+#PRISMA_FIELD_ENCRYPTION_KEY=k1.aesgcm256.RhSAzMU4Af5OKXUYQosqNLMRK1Nc3PicRp5iWa6I7Ko=
+
+
 NEXTAUTH_SECRET=dAbxJF2DRzqwGYn+BWKdj8o9ieMri4FWsmIRn77r2F8=
 REACT_APP_API_URL=http://localhost:3333/api
 NEXTAUTH_URL=http://localhost:3000
-NEXT_AUTH_API_URL=http://backend:3333/api
+NEXTAUTH_API_URL=http://backend:3333/api
+NEXT_PUBLIC_SIGNOUT_REDIRECT_URL=http://localhost:3000
+NEXT_PUBLIC_TRIAL_LOCK_PING_TIME_MS=240000
+NEXT_TRIAL_LOCK_CLEAR_TIME_MS=300000
+
+
+#keycloak settings
 KEYCLOAK_URL=<KEYCLOAK_URL>
 KEYCLOAK_CLIENT_ID=<client_id>
+# Keycloak client uuid. This is different from the id above.
+# If you cannot find it in the client settings, the URL usually includes it. Example: http://localhost:8080/admin/master/console/#/ctims/clients/3813811a-***/settings, 3813811a-*** is the uuid
 KEYCLOAK_CLIENT_UUID=<uuid>
+# We suggest creating a new realm instead of using the default master realm.
 KEYCLOAK_REALM=<realm>
+# This is the client secret that stored under the Credentials tab in the client page.
+# The realm needs to have client authentication enabled to see the Credentials tab.
 KEYCLOAK_CLIENT_SECRET=<client_secret>
-KEYCLOAK_ADMIN_CLIENT_ID=ctims-admin # a client with sevice account enabled
+# a client with sevice account enabled
+# It does not need to be a separate client. If your above client already has service account enabled, use the same ID here.
+KEYCLOAK_ADMIN_CLIENT_ID=ctims-admin 
 KEYCLOAK_ADMIN_CLIENT_SECRET=<secret>
-KEYCLOAK_TOKEN_ENDPOINT=https://cbioportal.pmgenomics.ca/newauth/realms/UHN/protocol/openid-connect/token
-NEXT_PUBLIC_ENABLE_MATCHMINER_INTEGRATION=true
-NEXT_PUBLIC_TRIAL_LOCK_PING_TIME_MS=240000
-NEXT_TRIAL_LOCK_CLEAR_TIME_MS=300000
+# This is the endpoint to get access token.
+KEYCLOAK_TOKEN_ENDPOINT=
+
+#matchminer settings - only if matchminer integration is enabled
+NEXT_PUBLIC_ENABLE_MATCHMINER_INTEGRATION = true
+MM_API_URL=<matchminer-api-url>/api
+MM_API_TOKEN=
+
+#Rabbitmq settings for MatchMiner communication
+RABBITMQ_URL="rabbitmq"
```

Comment:
do we need quotes here? should work without quotes, same with run_match and match_message